views/layout.jsx

```diff
@@ -2,24 +2,76 @@ import { createElement } from "complate-stream";
 
 let CSS = `
 body {
-	max-width: 40em;
+	margin: 0;
+	font-family: Trebuchet MS1, Trebuchet MS, sans-serif;
```

Comment:
https://www.cssfontstack.com/Trebuchet-MS

We don't have to use this font if someone has another suggestion. I mainly wanted to find a web-safe font that isn't Arial or Georgia to make the site look a _bit_ different. For now, I wanted to avoid a web font because of the extra bytes (although I personally am not anti-webfont). I basically just googled for web-safe fonts and chose one that I thought looked nice. Now I can't find the reference site, but according to https://www.w3schools.com/cssref/css_websafe_fonts.asp we should probably add `Helvetica` as a backup.

Reply:
Makes sense - it might be worth consulting @nanoparsec at some point, but this is fine with me.

Reply:
I think it would definitely be worth consulting with @nanoparsec about this. :)

Reply:
I found a notification in my inbox...Maybe you should ping me on Slack if you want faster responses. 😸 

What is the difference between Trebuchet MS1 and Trebuchet MS? I couldn't find anything about a Trebuchet MS1 but maybe I am blind.

However Trebuchet MS should be preinstalled on Windows, Mac and iOS.
Helvetica is preinstalled only on Macs https://www.cssfontstack.com/ so when adding that as a backup font you catch those Mac users who have no Trebuchet MS.

Reply:
Yes, I think I will change the font stack to be `Trebuchet MS, Helvetica, sans-serif` if we want to continue using this font.

@nanoparsec: I also wanted to ask you your opinion on the font in general. I picked it out very quickly because I liked it (in other words, I didn't put that much thought into it), but I am open to using another font.

Reply:
I think Trebuchet is a good choice if you want to avoid a web font and look a bit different than the standard font (Arial etc.)
I would keep that font for now and discuss that later if necessary :)

Reply:
Excellent. As the resident old guy, I think increased font size wouldn't hurt though.

Reply:
Good point. I will play around with some bigger font sizes.